defaults/main.yml

```diff
@@ -158,8 +157,9 @@ wimpy_user_data: |-
           [Service]
           User=core
           Restart=always
+          ExecStartPre=/usr/bin/docker pull {{ wimpy_docker_image_name }}:{{ wimpy_release_version }}
```

Comment:
This forces the download of the docker image before executing the healthcheck script